test/integration/tasks/create-new-enviromenment.js

```diff
@@ -0,0 +1,17 @@
+const nanoid = require('nanoid')
+const getCurrentSpace = require('../contentful-client').getCurrentSpace
+const printStepTitle = require('../utils').printStepTitle
+
+module.exports = async () => {
+  printStepTitle('Creating a new environment for testing')
+
+  const space = await getCurrentSpace()
+
+  const environmentId = nanoid()
+
+  await space.createEnvironmentWithId(environmentId, { name: environmentId }, 'master')
```

Comment:
Does it wait for creation to be completed? What do you get as `sys.status` right after the call is finished?

Reply:
fixed